src/bin/travis/testCentaurJes.sh

```diff
@@ -72,3 +73,5 @@ export CENTAUR_LOG_PATH="gs://cloud-cromwell-dev/cromwell_execution/travis/centa
 gsutil cp ${CENTAUR_LOG_PATH} centaur.log
 cat centaur.log
 echo "More logs for this run are available at https://console.cloud.google.com/storage/browser/cloud-cromwell-dev/cromwell_execution/travis/centaur/${WORKFLOW_ID}/call-centaur/"
+
+exit "${EXIT_CODE}"
```

Comment:
How are quotes dangerous? Not testing to see if a variable is set is dangerous, but not quoting to avoid accidental shell expansion.